docs/setup_instructions/openrrp-setup.md

```diff
@@ -0,0 +1,62 @@
+
+
+
+## Set up your accounts
+
+Before the workshop begins, you will need to create and configure your GitHub account.
+You will also need to set up your OpenScPCA Amazon Web Services (AWS) account.
+
+<!-- Relative links need to use html since this document is rendered via %include% -->
+1. [Create a GitHub account](github_gitkraken_setup_instructions.html#creating-a-github-account) if you do not already have one.
+    - You should also [turn on two-factor authentication (2FA)](github_gitkraken_setup_instructions.html#enabling-two-factor-authentication-2fa-on-github) after you create your account.
+
+2. Before the workshop, you will receive an email inviting you to join AWS IAM Identity Center, which will allow you to access OpenScPCA data and compute resources.
+    - [Follow these instructions](https://openscpca.readthedocs.io/en/latest/software-platforms/aws/joining-aws/) to accept this invitation and configure multifactor authentication.
+    - _You are only eligible for an OpenScPCA AWS account after you fill out the [OpenScPCA intake form](https://share.hsforms.com/1MlLtkGYSQa6j23HY_0fKaw336z0) where you agree to [OpenScPCA's policies](https://openscpca.readthedocs.io/en/latest/policies/)._
+
+## Workshop software installations
+
+You will also need to set up your computers following _some_ these software installation steps in the [OpenScPCA Project's setup instructions](https://openscpca.readthedocs.io/en/latest/technical-setup/), as detailed below.
+
+You should not complete all setup steps on the OpenScPCA website, since we will be doing some steps together as part of the workshop itself, but you should come prepared with all software installed and ready to be further set up.
+
+### Instructions for macOS users
+
+1. [Install GitKraken](https://openscpca.readthedocs.io/en/latest/technical-setup/install-a-git-client/#install-gitkraken), a GUI for working with Git
+    - Follow the instructions in the link above to also sign into GitKraken with your GitHub account
+    - [See these instructions](github_gitkraken_setup_instructions.html#setting-up-gitkraken) for additional information about setting up GitKraken
+2. [Install conda](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/setup-conda/#install-conda), the software we use to manage software dependencies in OpenScPCA.
+    - You do _not_ need to also set up conda; we will do this together during the workshop
+3. [Install R and RStudio](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/install-r-rstudio/#install-r-on-macos)
+    - Follow the instructions to install the additional tools (XCode developer tools and the GNU Fortran compiler) as well.
```

Comment:
See https://github.com/AlexsLemonade/reproducible-research/pull/189#discussion_r1755261895, but I'm not sure how much else might need to change
```suggestion
    - Follow the instructions to install the additional tools (XCode command line tools and the GNU Fortran compiler) as well.
```